adviser/overlays/moc-prod/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+---
+kind: ConfigMap
```

Comment:
we would need to add this to the kustomize file as well.
please keep this under the patchstrategymerge:
https://github.com/thoth-station/thoth-application/blob/a115c78b4a32841e7f9e259a45bc4b2232174d08/adviser/overlays/moc-prod/kustomization.yaml#L10